src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const ul = document.querySelector('.gallery__list');
+const div = document.querySelector('.gallery__large-img ');
```

Comment:
There is an extra space at the end of the query selector string. This might cause issues when trying to select the element. Please remove the space after 'gallery__large-img'.